ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/recaptcha/v1/recaptcha/_cq_dialog/.content.xml

```diff
@@ -18,6 +18,7 @@
         jcr:primaryType="nt:unstructured"
         jcr:title="Adaptive Form reCAPTCHA"
         sling:resourceType="cq/gui/components/authoring/dialog"
+        extraClientlibs="[core.forms.components.recaptcha.v1.editor]"
```

Comment:
Can you update the recaptcha README.md with this new feature ?

Reply:
Done